### PR TITLE
Topoplot series, fresh base

### PR DESCRIPTION
**Pull request by behinger**
this replaces #2, now with better history.

Also docstring + tests for the binning added.

Docstring for eeg_topoplot_series (rename!) was added as well + example

- [x] add eeg_topoplot_series testcase

### CODE REVIEW

**Comment by behinger** [comment]
I discussed with Phillip last week. He is greatly in favour of removing AlgebraOfGraphics.jl as a dependency.

To do so, we'd need to set out the grid ourselfs.
I think the following two modes are important:

row-wise with multiple rows possible
```
1 2 3 4 5 6
A B C D E F
```
continuous
```
1 2 3 4
5 6 7 8
```

**Comment by behinger** [comment]
hey - if this is not the right place for this tool, then I can simply include it in UnfoldMakie.jl - I just need to know :)

**Comment by SimonDanisch** [comment]
No, I think this is the right place! :) Just didn't have a lot of time to review before the basics weren't working...
Will try to review today!

File: src/TopoPlots.jl [hunk 1]
```diff
@@ -12,6 +12,11 @@ using Delaunay
 using Dierckx
 using ScatteredInterpolation
 
+# for topoplot_series
+using DataFrames
+using CategoricalArrays
+
```
[the remark this answers is not in the document]
Comment (behinger) [reply src/TopoPlots.jl:18]: we can remove DataFrames, but I need CategoricalArrays for the `cut` function

[the remark this answers is not in the document]
Comment (behinger) [reply src/TopoPlots.jl:18]: I checked, it wont be easy to remove all of DataFrames functionality (most importantly the functionality of `stack`),

**Comment by behinger** [comment]
Hi!
there is really no issue to take this into UnfoldMakie.jl - but this is now open for review for ~3 month...

Should I just take it to UnfoldMakie.jl until this is reviewed? I will do this tomorrow. I can always remove it again when this is merged.

**Comment by palday** [comment]
@behinger the big blocker at the moment is moving away from the DataFrames-based API. I can do this rewrite, but don't have time at the moment. So I guess put it in UnfoldMakie and when we get this integrated here, then you can change UnfoldMakie to just call the functionality here. Thanks for the contribution and sorry for the delay!

**Pull request closed without merging**